gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -412,6 +414,16 @@
   public static final HadoopConfigurationProperty<RedactedString> GCS_ENCRYPTION_KEY_HASH =
       new HadoopConfigurationProperty<>("fs.gs.encryption.key.hash");
 
+  /** Configuration for authorization handler. */
+  public static final HadoopConfigurationProperty<Class<? extends AuthorizationHandler>>
+      GCS_AUTHORIZATION_HANDLER_IMPL =
+          new HadoopConfigurationProperty<>("fs.gs.authorization.handler.impl");
+
+  /** Configuration prefix for custom authorization hook properties. */
+  public static final HadoopConfigurationProperty<Map<String, String>>
+      GCS_AUTHORIZATION_HANDLER_PROPERTY_PREFIX =
+          new HadoopConfigurationProperty<>("fs.gs.authorization.properties.", ImmutableMap.of());
```

Comment:
Rename to `fs.gs.authorization.handler.properties.` to be more explicit and also add this property to release notes and configuration doc.